articles/constants-and-variables.html

```diff
@@ -842,13 +843,19 @@ <h4>Each Local Declared Variable Must Be Used At Least Once Effectively</h4>
 }
 </code></pre>
 
+<p>
+You should remember that above fix is <b>not a good solution</b>. It would be better
+to investigate your code and remove useless variables. This would help
+<b>readability</b> of your code.
+</p>

```

Comment:
Yes, I also think here needs more explanations. 
On the other hand, I recommend to use the following one instead.

```
Generally, the above fix is not recommended to be used in production code.
It should be used on development and debug phases only.
Cleaning up unused variables are both good for code readability and execution performance.
```

Reply:
In general, I find it bad habit to have `unused` variables in the code by force. It will make code cleaning and readability very hard, and newbie programmer may even get use to it and do it all the time. Don't you agree?

Reply:
I agree. I just found there are some grammar mistakes in my suggested version. So here I would further improve it as:
```
Generally, the above fix is not recommended to be used in production code.
It should be used inn development and debug phases only.
It is not a good habit to leave unused local variables in code,
for unused local variables have negative effects on both code readability and program execution performance.
```